src/locate/controllers/locate.py

```diff
@@ -0,0 +1,157 @@
+from flask import Blueprint, request, jsonify
+from helpers import helper
+import sys
+import ast
```

Comment:
Same here...changes in the netmanager microservice yet this PR is for `location-registry` 🤔